views/songs/language.pug

```diff
@@ -24,12 +42,28 @@ block content
             button.btn.btn-default Add
 
     script(type="text/javascript").
-        function editLanguage (code) {
-          $('#' + code + ' .display').hide();
-          $('#' + code + ' .editor').show();
+        function editLanguage (length, code) {
+          let count = 0;
```

Comment:
A simpler way to get the effect you want:
1. Add a class='language' attribute to the language li
2. Hide all of the language elements $('.language').hide();
3. Show the one that is being edited $('#' + code).show();
4. make it editable 
$('#' + code + ' .display').hide();
$('#' + code + ' .editor').show();